generatorreceiver/internal/topology/service_route.go

```diff
@@ -2,13 +2,14 @@ package topology
 
 import (
 	"fmt"
-	"github.com/lightstep/demo-environment/generatorreceiver/internal/flags"
 	"math/rand"
+
+	"github.com/lightstep/demo-environment/generatorreceiver/internal/flags"
```

Comment:
very tiny nit: I think at Lightstep we tend to alphabetize imports, so you should move this import to after `fmt` (and delete the empty line while you're at it 😄 )

Reply:
Never mind! I realized that we separate out standard packages from ones imported from github etc. So keep the empty line and the order you have!